libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Plutus/ScriptTestContext.hs

```diff
@@ -16,7 +17,7 @@ data PlutusArgs = PlutusArgs
   { paData :: Data
   , paSpendDatum :: Maybe Data
   }
-  deriving (Generic)
+  deriving (Generic, Show)
```

Comment:
The changes to `cardano-ledger-alonzo` don't actually depend on this change in `cardano-ledger-core`. They did when I first started this PR, but they don't now. I confirmed this by dropping the change in `-core` and rebuilding. So we don't need to change the version bound in `-alonzo`. I've left the change in the PR, though, because I think it's useful anyway.